text.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main>
+        <header class="header">
+            <div class="logo">Jo</div>
+            <div></div>
+            <div class="menu">
+                <a href="#" class="nav">
+                    <div ></div>
+                    <div ></div>
+                    <div ></div>
+                </a>
+            </div>
+        </header>
+        <section class="headline">
```

Comment:
- [ ] I can see that you have no background on your headline section which is present in the Figma design you chose.  I would recommend you add the background images to make your UI look great 👍🏽 .

 You can equally export the background images from Figma 😄 .  Kindly ensure your design match the Figma design you chose👍🏽. 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/191638473-dec0ce28-07c6-4cc0-a31e-d605e9c23bb3.png)  | ![image](https://user-images.githubusercontent.com/63473584/191638504-845963f6-0544-4a42-a721-3bcd83a49f9f.png)  |